src/utils/fetchLocalAuthorities.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+export default async () => {
+  // const url = 'https://www.planning.data.gov.uk/entity.json?dataset=local-authority&limit=500'
+  const url = 'https://datasette.planning.data.gov.uk/digital-land.json?sql=select%0D%0A++distinct+provision.organisation%2C%0D%0A++organisation.name%2C%0D%0A++organisation.dataset%0D%0Afrom%0D%0A++provision%2C%0D%0A++organisation%0D%0Awhere%0D%0A++provision.organisation+%3D+organisation.organisation%0D%0Aorder+by%0D%0A++provision.organisation'
```

Comment:
Can you pull the SQL as a plain string and url encode it (`encodeURIComponent`)? It will be easier to read than this.